components/x-teaser/src/Image.jsx

```diff
@@ -17,20 +17,24 @@ const aspectRatio = ({ width, height }) => {
 	return null;
 };
 
-const NormalImage = ({ src }) => (
-	<img className="o-teaser__image" src={src} alt="" />
+const NormalImage = ({ src, srcSet }) => (
+	<img className="o-teaser__image" src={src} srcSet={srcSet} alt="" />
 );
 
-const LazyImage = ({ src, lazyLoad }) => {
+const LazyImage = ({ src, srcSet, lazyLoad }) => {
 	const lazyClassName = typeof lazyLoad === 'string' ? lazyLoad : '';
-	return <img className={`o-teaser__image ${lazyClassName}`} data-src={src} alt="" />;
+	return <img className={`o-teaser__image ${lazyClassName}`} data-src={src} srcSet={srcSet} alt="" />;
 };
 
-export default ({ relativeUrl, url, image, imageSize, imageLazyLoad, ...props }) => {
+export default ({ relativeUrl, url, image, imageSize, imageLazyLoad, showHiResImages, ...props }) => {
 	const displayUrl = relativeUrl || url;
 	const useImageService = !(image.url.startsWith('data:') || image.url.startsWith('blob:'));
 	const imageSrc = useImageService ? imageService(image.url, ImageSizes[imageSize]) : image.url;
 	const ImageComponent = imageLazyLoad ? LazyImage : NormalImage;
+	const srcSet = showHiResImages && useImageService ? [
+		imageService(image.url, ImageSizes[imageSize], null, {dpr:1}),
```

Comment:
Is dpr=1 actually needed? Wouldn't the browser use the `src` attribute if the screen density isn't high enough? Or does the browser need this for some other reason?

Also, have you had a look to see it's worth also altering compression/quality settings in addition to the dpr?

Reply:
I changed this as it seems that dpr=2 was used to cover hi-dpi and downscaled via CSS for normal size... I presumed it might help performance not to downscale but also using the image service as intended. A nice side-effect has been it makes it helps to see when the browser is using the default image size and not the hi-res.

from image service:
> Device Pixel Ratio, i.e. number of image pixels per CSS pixel. The default is 1, which gives the standard 1:1 ratio of image pixels to CSS pixels. Set to 2 for high-DPI ("Retina") displays. Request to the image service with width=100&dpr=2 will output image 200 pixels wide, but optimized to be displayed with CSS width:100px

Reply:
It's `dpr:2` by default in the image service handler, but this is scaled down for non-hi-density displays, with the intention of covering both, I think.
By setting `dpr:1` we should be downloading smaller images for non-hi-density displays and easing the load on the browser caused by this scaling
This behavour is recommended by the image service:
> Device Pixel Ratio, i.e. number of image pixels per CSS pixel. The default is 1, which gives the standard 1:1 ratio of image pixels to CSS pixels. Set to 2 for high-DPI ("Retina") displays. Request to the image service with width=100&dpr=2 will output image 200 pixels wide, but optimized to be displayed with CSS width:100px

Note that browsers don't automatically adjust intrinsic size of images with higher DPI, so you need'll need to use some "responsive image" solution, e.g. set explicit width or height, use image-set() or srcset attribute.

Unless this is intended to increase the quality of the image via the scaling?

Reply:
reverted to dp=2 for all (set 2x to use highest images quality)